use advent_of_code::*;
use std::error::Error;

fn main() -> Result<(), Box<dyn Error>> {
    run_advent_problem::<Naive, aoc_2023::if_you_give_a_seed_a_fertilizer::IfYouGiveASeedAFertilizer>(
    )
}
